runtime/package.json

```
@@ -22,34 +22,35 @@
   },
   "dependencies": {
     "@babel/polyfill": "^7.8.3",
-    "@expo/babel-standalone": "1.0.3",
     "@expo/vector-icons": "^12.0.0",
     "@react-native-async-storage/async-storage": "~1.15.0",
     "diff": "^5.0.0",
     "escape-string-regexp": "^5.0.0",
     "expo": "^44.0.0",
     "expo-analytics-amplitude": "~11.1.0",
     "expo-app-loading": "~1.3.0",
-    "expo-asset": "~8.4.5",
+    "expo-asset": "~8.4.6",
     "expo-barcode-scanner": "~11.2.0",
-    "expo-constants": "~13.0.0",
-    "expo-file-system": "~13.1.0",
+    "expo-constants": "~13.0.1",
+    "expo-file-system": "~13.1.4",
     "expo-font": "~10.0.4",
     "expo-keep-awake": "~10.0.1",
     "expo-random": "~12.1.1",
     "expo-status-bar": "~1.2.0",
-    "expo-updates": "~0.11.3",
+    "expo-updates": "~0.11.6",
     "path": "^0.12.7",
     "prop-types": "^15.7.2",
     "pubnub": "^4.23.0",
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "react-native": "0.64.3",
     "react-native-gesture-handler": "~2.1.0",
-    "react-native-reanimated": "~2.3.1",
+    "react-native-reanimated": "~2.3.3",
+    "react-native-reanimated-babel-standalone": "npm:react-native-reanimated@~2.3.1",
```

```suggestion
    "react-native-reanimated-babel-standalone": "npm:react-native-reanimated@~2.3.3",
```

should these be in sync?